### ISSUE TITLE
Add support for `ogg` file while inserting RTE Media

### ISSUE THREAD

**Issue by phuongnq** [craftercms/craftercms#6535]
### Latest version

- [X] The issue is in the develop branch
- [X] The issue is in the latest released 4.1.x
- [ ] The issue is in the latest released 4.0.x
- [ ] The issue is in the latest released 3.1.x

CI:4706

### Duplicates

- [X] I have searched the existing issues

### Is your enhancement request related to a problem? Please describe.

RTE Media plugin has support for audio files with extensions such as `.ogg` but in CrafterCMS users cannot select/upload `.ogg` file while inserting Media in the RTE.

### Describe the solution you'd like

Please add support for `.ogg` files while inserting Media in the RTE.

**Comment by rart** [comment]
@russdanner The `.ogg` extension can be used for both audio and video. We don't have an audio datasource. Should we create one? Otherwise, using a video datasource would display mp4s and other video extensions while selecting.

In the case of `ogg` files, in an audio datasource, I'm not sure there would be a way of distinguishing between ogg video vs ogg audio. Must be researched to try to display only relevant files on the file pickers.

**Comment by rart** [comment]
@sumerjabri @russdanner 

- Adding `.ogg` to the video datasource would be a sufficient solution to allow for the ogg audio files to be used on RTE media on form engine 1. FE2 would have the audio datasource. The down-sides are:
  - If audio is all that's desired, the video datasource file picker won't disallow picking non audio-only files
  - The content modeller must think to go this route (using video datasource to select ogg)

- Adding an audio file datasource would solve the above, but would require: 
  - UM to include the datasource into blueprints
  - Updating our maintained blueprints
  - Updating docs(?)
  - Alternatively we could create the datasource but leave it unattached from blueprints; consumers would need to add it if/when required.

- An intermediate approach would be to modify the video datasource to become a "media" datasource, and have checkboxes for allow video and allow audio. 

There's a number of video datasources: Video From WebDav Repository, Video Uploaded to WebDav Repository, Video From S3 Repository, Video Uploaded to S3 Repository, Video Transcoding From S3 Repository, Video Uploaded From Desktop, Video From Repository. I assuming the scope of this ticket is "Video Uploaded From Desktop" only.

**Comment by sumerjabri** [comment]
@rart, let's create an audio data source and then assign it to BE for UM work.

**Comment by rart** [comment]
@jvega190 👆

**Comment by jvega190** [comment]
The configuration file that needs to be updated is `site-config-tools.xml`, under `datasources`
```
<datasource>
  <name>audio-desktop-upload</name>
  <icon>
    <class>fa-volume-up</class>
  </icon>
</datasource>
<datasource>
  <name>audio-browse-repo</name>
  <icon>
    <class>fa-file-audio-o</class>
  </icon>
</datasource>
```

**Comment by phuongnq** [comment]
Verified that we can use Audio OGG file as RTE Media when using the data source:
```
<datasource>
  <name>audio-desktop-upload</name>
  <icon>
    <class>fa-volume-up</class>
  </icon>
</datasource>
<datasource>
  <name>audio-browse-repo</name>
  <icon>
    <class>fa-file-audio-o</class>
  </icon>
</datasource>
```

**Comment by phuongnq** [comment]
@sumerjabri I noticed this ticket is marked as test and validate, but do we want to add this to the UM and have this by default, and for built-in blueprints?

**Comment by yacdaniel** [comment]
@rart @jvega190 
please add a proper audio icon instead of the `?` that shows in the Sidebar for the new audio datasource files. 
![image](https://github.com/craftercms/craftercms/assets/6722074/e1d37f50-6e03-4f69-a4b0-edfb9a2104bc)

**Comment by yacdaniel** [comment]
Verified fixed in `4.1.4`

**Comment by yacdaniel** [comment]
Verified in `4.2.0-SNAPSHOT
